rs/nns/governance/canbench/canbench_results.yml

```diff
@@ -55,7 +55,7 @@ benches:
     scopes: {}
   compute_ballots_for_new_proposal_with_stable_neurons:
     total:
-      instructions: 2169168
+      instructions: 2220000
```

Comment:
canbench is too sensitive. This is being [fixed] as we speak tho.

[fixed]: https://github.com/dfinity/canbench/pull/79